openc3/lib/openc3/microservices/decom_microservice.rb

```diff
@@ -95,6 +95,11 @@ def decom_packet(topic, msg_id, msg_hash, _redis)
         packet.stored = ConfigParser.handle_true_false(msg_hash["stored"])
         packet.received_time = Time.from_nsec_from_epoch(msg_hash["received_time"].to_i)
```

Comment:
We set received_time here but not packet_time?

Reply:
Probably